mungegithub/publisher/deployment/kubernetes/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubernetes-publisher-config
+data:
+  # basic config options.
+  # TODO: uncomment this before merge
+  #  config.organization: kubernetes
+  config.organization: caesarxuchao
+  config.project: kubernetes
+  config.pr-mungers: publisher
+  config.token-file: /etc/secret-volume/token
+
+  # munger specific options.
+  gitrepo.repo-dir: /gitrepos
+  config.period: "24h"
```

Comment:
A challenge with this is that it's going to list all open PRs and use some github api tokens for no reason.